pod/perldelta.pod

```diff
@@ -27,21 +27,37 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 Subroutine signatures are no longer experimental
+
+Introduced in Perl version 5.20.0, and modified several times since, the
+subroutine signatures feature (C<use feature 'signatures'>) is now no longer
+considered experimental. It is now considered a stable language feature and
+is included in the C<:5.36> feature bundle, enabled automatically by
+C<use v5.36>, and no longer prints a warning.
+
+    use v5.36;
+
+    sub add ($x, $y) {
+      return $x + $y;
+    }
+
+Despite this, certain elements of signatured subroutines remain
+experimental; see below.
+
 =head2 @_ is now experimental within signatured subs
 
-Even though subroutine signatures currently remain experimental, use of the
-default arguments array (C<@_>) with a subroutine that has a signature is
-specifically also experimental, with its own warning category.  Silencing the
-C<experimental::signatures> warning category is not sufficient to dismiss
-this.  The new warning is emitted with the category name
-C<experimental::args_array_with_signatures>.
+Even though subroutine signatures are now stable, use of the default arguments
+array (C<@_>) with a subroutine that has a signature remains experimental,
+with its own warning category.  Silencing the C<experimental::signatures>
+warning category is not sufficient to dismiss this.  The new warning is
+emitted with the category name C<experimental::args_array_with_signatures>.
 
 Any subroutine that has a signature and tries to make use of the defaults
 argument array or an element thereof (C<@_> or C<$_[INDEX]>), either
 explicitly or implicitly (such as C<shift> or C<pop> with no argument) will
 provoke a warning at compile-time:
 
-    use experimental 'signatures';
+    use v5.36;
```

Comment:
Was this removed line a typo instead of saying `use feature 'signatures'`?

Reply:
No, it's intentional. The general theme here is that as much as possible we should be using/encouraging `use v5.36` as a general way to turn on signatures (and other features). I thought I might as well tidy up the perldelta entry about this related change while we're here.

Reply:
That's not what I meant.  What I meant was that the old version looked like a typo for saying `experimental` rather than `feature`, and I was confirming this.  I agree that we want `use v5.36` now.

Reply:
It was not a typo. experimental is a recommended module for enabling experimental features.